perf_dashboard/README.md

```diff
@@ -0,0 +1,43 @@
+# Istio Performance Dashboard
+
+## How to run this application locally
+
+- Install Docker on your local machine.
+    - [MacOS](https://docs.docker.com/docker-for-mac/install/)
+    - [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/#os-requirements)
+    - [Windows](https://docs.docker.com/docker-for-windows/install/)
+
+- Make sure Docker compose installed on you local machine.
```

Comment:
we shouldn't use docker compose, everything else in the istio project uses kubernetes, and this will likely be deployed on k8s eventually. But fine for now/dev

Reply:
Sure, I can update this part later on.